filters/max_length.py

```diff
@@ -1,18 +1,36 @@
 #!/usr/bin/env python3
 from sys import stdin, stdout, stderr
 import argparse
-
+import re
 
 def parse_user_args():
     """Parse the arguments necessary for this filter"""
     parser = argparse.ArgumentParser(description="Filters a parallel or mono dataset based on line lengths")
     parser.add_argument("--max-length", default=150, type=float)
     parser.add_argument("--min-length", default=1, type=float)
+    parser.add_argument("--count-type", default='words', type=str)
```

Comment:
You can add `choices=['words', 'chars', 'bytes']` here to make the `--help` output a bit more clear.